### PR TITLE
chore: adding SGTstake Bootstrap Node

### PR DESCRIPTION
**Pull request by SGTstake**
## Description

adding SGTstake Bootstrap Node.

### CODE REVIEW

File: config/bootstrap.json [hunk 1]
```diff
@@ -65,4 +65,10 @@
         "website": "https://IONode.Online",
         "address": "/dns/pactus-bootstrap.ionode.online/tcp/21888/p2p/12D3KooWG5tvvfS8PRd2Bgyxxqrok7QfohkHSprFPxmT3HTh8dVD"
     }
+    {
+        "name": "SGTstake",
+        "email": "adorid@sgtstake.com",
+        "website": "https://sgtstake.com",
+        "address": "/dns/159.148.146.149/tcp/21888/p2p/12D3KooWKCokWtpdudxgsLRoFcnrW35vhn6w632iGWCga7E5e68Q"
```
Comment (kehiy) [inline config/bootstrap.json:72]: You should use `ip4` here instead of `dns` since it's not an DNS address (domain name).

Comment (SGTstake) [reply config/bootstrap.json]: fixed)

Comment (kehiy) [reply config/bootstrap.json]: Thanks.

**Review by b00f — APPROVED** [review]

**Comment by b00f** [comment]
@SGTstake Thanks

**Pull request merged**